core/templates/components/shared-component.module.ts

```diff
@@ -73,6 +73,13 @@ import { SummarizeNonnegativeNumberPipe } from 'filters/summarize-nonnegative-nu
 import { SortByPipe } from 'filters/string-utility-filters/sort-by.pipe';
 import { FilterForMatchingSubstringPipe } from 'filters/string-utility-filters/filter-for-matching-substring.pipe';
 import { WrapTextWithEllipsisPipe } from 'filters/string-utility-filters/wrap-text-with-ellipsis.pipe';
+import { PromoBarComponent } from './common-layout-directives/common-elements/promo-bar.component';
+import { DynamicContentModule } from './angular-html-bind/dynamic-content.module';
+import { ObjectComponentsModule } from 'objects/object-components.module';
+import { OnScreenKeyboardComponent } from './on-screen-keyboard/on-screen-keyboard.component';
+import { TranslateModule } from 'filters/translate.module';
+import { SharedFormsModule } from './forms/shared-forms.module';
+import { SideNavigationBarComponent } from './common-layout-directives/navigation-bars/side-navigation-bar.component';
```

Comment:
```suggestion
import { 
  SideNavigationBarComponent 
} from './common-layout-directives/navigation-bars/side-navigation-bar.component';
```

Reply:
exceding 80 length limit this way.
![image](https://user-images.githubusercontent.com/24855641/115417686-70c6f780-a216-11eb-8676-a3a698326df3.png)

Reply:
Hmm, but your original code also exceeds 80 chars right?

Reply:
We actually prefer import statements to be on the same line.

Reply:
import statements are excluded from 80 length rule.

Reply:
@nithusha21 Hmm, okay. Do we have any reason for this? 
@ashutoshc8101 Btw, fine to keep it as is in this PR.